File: FirstBitSolutions/C Module/Assignments/Assignment 3/LoopsToNested/Ques6.c
//6 Check the given number is Perfect number or not.
//Input: n = 28
//Output: Perfect
#include<stdio.h>
void main() {
	int n,sum,i;
	n = 28;
	sum = 0;

	for(i=1; i<n; i++) {
		if(n%i==0) {
			sum =sum+i;
		}
	}
	if(sum==n) {
		printf("%d is Perfect number",n);
	} else {
		printf("%d is not Perfect number",n);
	}
}

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 3/Ques6.java
class Ques{ 
public static void main(String args[]){ 

	int n,sum,i;
	i = 1;
	n = 28;
	sum = 0;
	
	while(i<n){
		if(n%i==0){
			sum +=i;
		}	
		i++;
	}
	if(sum==n){
		System.out.printf("%d is Perfect number",n);
	}else{
		System.out.printf("%d is not Perfect number",n);	
	}
 }
}